RandomData.Api/Helpers/SwashbuckleHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
+
+namespace RandomData.Api.Helpers
```

Comment:
Noooo, "helpers" are almost always bad idea. Sounds like everything. Entire application helps with something. Most of the time static classes like this end with just Extensions post fix.